guides/common/modules/proc_building-a-discovery-image.adoc

```diff
@@ -44,6 +41,18 @@ $ cd foreman-discovery-image
 ----
 
 .Procedure
+. Replace content of `00-repos-centos8.ks` with the following content:
```

Comment:
```suggestion
. Replace content of the `00-repos-centos8.ks` file with the following text:
```

Reply:
I went with a slightly different text. How about now?

Reply:
Looks good to me!